# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/mp3info/hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN -x '*~' mp3info-0.8.4/gmp3info.c mp3info-0.8.4-gcc-3.3.1/gmp3info.c
--- mp3info-0.8.4/gmp3info.c	2001-07-16 11:17:16.000000000 +0200
+++ mp3info-0.8.4-gcc-3.3.1/gmp3info.c	2003-08-19 22:49:53.000000000 +0200
@@ -79,8 +79,6 @@
 	        mp3.id3.genre[0]=gget_genre(tmp);
 	   	strcpy(tmp,gtk_entry_get_text(GTK_ENTRY(id3win_text_track)));
 		mp3.id3.track[0]=atoi(tmp);
-		if (mp3.id3.track[0] > 255)
-			mp3.id3.track[0]=255;
 	        write_tag(&mp3);
 	   }
 
@@ -484,12 +482,12 @@
 }
 
 void about_mp3info(void) {
-	quick_popup(VERSION,"MP3Info
-			An ID3 Tag Editor\n\n
-			Original author: Ricardo Cerqueira <rmc@plug.pt>
-			Current maintainer: Cedric Tefft <cedric@earthling.net>
-			GTK Version: Ricardo Cerqueira <rmc@plug.pt>\n\n
-			Ricardo Cerqueira, Cedric Tefft   1999-2001\n\n");
+	quick_popup(VERSION,"MP3Info\n"
+" 			An ID3 Tag Editor\n\n"
+"			Original author: Ricardo Cerqueira <rmc@plug.pt>\n"
+"			Current maintainer: Cedric Tefft <cedric@earthling.net>\n"
+"			GTK Version: Ricardo Cerqueira <rmc@plug.pt>\n\n"
+"			Ricardo Cerqueira, Cedric Tefft   1999-2001\n\n");
 }
 
 /* rmcc has left the building */
